forge-gui/res/cardsfolder/upcoming/bind_to_secrecy.txt

```diff
@@ -10,8 +10,8 @@ SVar:SpendAnyMana:Mode$ Continuous | Affected$ Card.Self | EffectZone$ All | Aff
 SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
 SVar:Update:Mode$ ChangesZone | Origin$ Any | Destination$ Any | Static$ True | ValidCard$ Card.IsRemembered | Execute$ DBUpdate
 SVar:DBUpdate:DB$ UpdateRemember
-SVar:TrigDraft:DB$ NameCard | ConditionCheckSVar$ X | ConditionSVarCompare$ GE5 | Draft$ True | Defined$ You | ChooseFromList$ Corpse Churn,Corpse Hauler,Courier Bat,Durable Coilbug,Fear of Death,Gorging Vulture,Locked in the Cemetery,Naga Oracle,Necrotic Wound,Obsessive Stitcher,Reassembling Skeleton,Strategic Planning,Unmarked Grave,Wonder | SubAbility$ DBMakeCard
+SVar:TrigDraft:DB$ NameCard | ConditionCheckSVar$ X | ConditionSVarCompare$ GE5 | Draft$ True | Defined$ You | ChooseFromList$ Corpse Churn,Corpse Hauler,Courier Bat,Durable Coilbug,Fear of Death,Gorging Vulture,Locked in the Cemetery,Naga Oracle,Necrotic Wound,Obsessive Stitcher,Reassembling Skeleton,Strategic Planning,Unmarked Grave,Wonder | SubAbility$ DBMakeCard | SpellDescription$ If there are five or more mana values among cards in your graveyard, draft a card from CARDNAME's spellbook.
```

Comment:
@Hanmac even with this SpellDescription, Forge doesn't do anything after the Charm builder... any ideas?

![image](https://user-images.githubusercontent.com/103371817/173713345-6ddbaa83-6c5a-4b36-b984-7e0550a48dc5.png)

Fixing this probably needs a different PR